torch_geometric/nn/models/revgnn.py

```diff
@@ -0,0 +1,287 @@
+import copy
+from abc import ABC, abstractmethod
+from typing import Optional, Union
+
+import numpy as np
+import torch
+from torch import Tensor, nn
+
+from torch_geometric.typing import Adj
+
+
+class InvertibleFunction(torch.autograd.Function):
+    """Invertible autograd function."""
+    @staticmethod
+    def forward(ctx, fn, fn_inverse, num_bwd_passes, num_inputs,
```

Comment:
Will be great to add some brief documentation to explain these input arguments and the expected formats

Reply:
Added docs.